### ISSUE TITLE
[Feature Request]: Hide new videos from channels

### ISSUE THREAD

**Issue by geekdiop** [FreeTubeApp/FreeTube#4793]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Hello there, 

Can you add an option/button that allows the user to mute/hide a channel. By muting a channel, the user will not see new videos from "that channel" in the "Video" tab. Therefore, he has to go to the channel itself to see the new videos. 

Thanks.

### Proposed Solution

Having a button (next to SUBSCRIBE/UNSUBSCRIBE button) that tells the user: block/unblock the channel to see or hide its news videos. 

### Alternatives Considered

None

### Issue Labels

ease of use improvement, new feature

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I will be closing this as this is part of another issue. If https://github.com/FreeTubeApp/FreeTube/issues/4803 and https://github.com/FreeTubeApp/FreeTube/pull/4380 there will be no need for muting individual channels on the subscription page because the user can decide whether they want to see the latest or not
